scss/_variables.scss

```diff
@@ -1230,73 +1231,94 @@ $nav-pills-link-active-color:       $component-active-color !default;
-$navbar-active-transition:             bottom $navbar-transition-duration $navbar-transition-timing-function !default;
-// End mod
+
+$navbar-nav-link-padding-y:         1rem !default; // Boosted mod
+$navbar-nav-link-padding-x-xs:      $spacer * .25 !default; // Boosted mod
+$navbar-nav-link-padding-x:         $spacer * .5 !default;
 
 // Boosted mod: Orange navbar
 // scss-docs-start orange-navbar-variables
-$navbar-orange-supra-padding-x:               .125rem !default;
-$navbar-orange-supra-border-bottom-width:     1px !default;
-
-$navbar-orange-supra-badge-padding-y:         .125rem !default;
-$navbar-orange-supra-badge-padding-x:         .375rem !default;
-$navbar-orange-supra-badge-margin-top:        map-get($spacers, 4) * .1 !default;
-
-$navbar-orange-supra-icon-padding-top:        divide($spacer, 2) !default;
-$navbar-orange-supra-icon-padding-x:          .9375rem !default;
-$navbar-orange-supra-icon-padding-bottom:     .25rem !default;
-$navbar-orange-supra-icon-size:               1.5625rem !default;
-
-$navbar-orange-supra-link-padding-top:        .75rem !default;
-$navbar-orange-supra-link-padding-x:          map-get($spacers, 1) * 1.5 !default;
-$navbar-orange-supra-link-padding-bottom:     .6875rem !default;
+$navbar-transition-duration:              $transition-duration !default;
+$navbar-transition-timing-function:       $transition-timing !default;
+$navbar-transition:                       padding-top $navbar-transition-duration $navbar-transition-timing-function, padding-bottom $navbar-transition-duration $navbar-transition-timing-function, $transition-focus !default;
+$navbar-brand-transition:                 margin $navbar-transition-duration $navbar-transition-timing-function, $transition-focus !default;
+$navbar-brand-logo-transition:            width $navbar-transition-duration $navbar-transition-timing-function, height $navbar-transition-duration $navbar-transition-timing-function !default;
+$navbar-active-transition:                bottom $navbar-transition-duration $navbar-transition-timing-function !default;
+
+$navbar-border-width:                     $border-width * .5 !default;
+$navbar-border-color:                     $gray-700 !default;
+
+$navbar-brand-margin-end:                 $spacer * 1.5 !default;
+
+$navbar-brand-margin-y-xs:                $spacer * .5 !default;
+$navbar-brand-logo-size-xs:               $spacer * 1.5 !default;
+$navbar-brand-font-size-xs:               1.3125rem !default;
+$navbar-brand-letter-spacing-xs:          $letter-spacing-base * 5 !default;
+$navbar-brand-font-size-two-line-xs:      1.0625rem !default;
+$navbar-brand-letter-spacing-two-line-xs: $letter-spacing-base * 4 !default;
+
+$navbar-brand-margin-y:                   $spacer * .95 !default;
+$navbar-brand-logo-size:                  $spacer * 2.5 !default;
+$navbar-brand-font-size:                  2.1875rem !default;
```

Comment:
This one should be in `scss-docs-start navbar-variables`. Check that all variables from Bootstrap stays at their place.

Reply:
Doesn't seem to work to have `// scss-docs-start orange-navbar-variables` inside `// scss-docs-start navbar-variables`. In http://localhost:9001/docs/5.1/components/navbar/#sass-variables we have both set of variables.